dm/master/etcd.go

```diff
@@ -59,3 +68,130 @@ func startEtcd(masterCfg *Config,
 	}
 	return e, nil
 }
+
+// prepareJoinEtcd prepares config needed to join an existing cluster.
+// learn from https://github.com/pingcap/pd/blob/37efcb05f397f26c70cda8dd44acaa3061c92159/server/join/join.go#L44.
+//
+// when setting `initial-cluster` explicitly to bootstrap a new cluster:
+// - if local persistent data exist, just restart the previous cluster (in fact, it's not bootstrapping).
+// - if local persistent data not exist, just bootstrap the cluster.
```

Comment:
```suggestion
// - if local persistent data not exist, just bootstrap the cluster as a new cluster
```

Reply:
done in https://github.com/pingcap/dm/pull/350/commits/c9a413fe473ee766934ff1a2f045c49e7d63ebf0